### DISCUSSION TITLE
请问如果我在页面引入图片，图片的路径是啥样的，应该从哪里开始算，根路径是从哪里

### DISCUSSION THREAD

**Discussion by old-zsh** [umijs/umi#9864] [Q&A]
请问如果我在页面引入图片，图片的路径是啥样的，应该从哪里开始算，根路径是从哪里

**Comment by fz6m** [comment]
直接 import 即可拿到 url ，然后放入需要使用的位置即可。

```tsx
import url from './some.png'

// use
<img src={url} />
```

无需关心路径问题。打包完成后，静态资源将从 `publicPath` （默认为 `/` ）开始加载，如需使用 cdn ，要调整这个值。
